packages/amplify-cli/src/domain/amplify-usageData/NoUsageData.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable class-methods-use-this */
-import { ICommandInput, IFlowData } from 'amplify-cli-shared-interfaces';
-import { IFlowReport } from 'amplify-cli-shared-interfaces/lib/amplify-cli-flow-reporter-types';
+import { ICommandInput, IFlowData } from '@aws-amplify/amplify-cli-shared-interfaces';
+import { IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```

Comment:
These should be combined into a single import

Reply:
yes
```suggestion
import { ICommandInput, IFlowData, IFlowReport } from '@aws-amplify/amplify-cli-shared-interfaces';
```